docs/docs/configuration/animations.mdx

```diff
@@ -138,22 +173,21 @@ The default configuration is defined here: <a href="https://github.com/chartjs/C
 
 Direct property configuration overrides configuration of same property in a collection.
 
-These defaults can be overridden in `options.animation` or `dataset.animation` and `tooltip.animation`. These keys are also [Scriptable](../general/options.md#scriptable-options)
+To enable a custom collection, the default one of the same type must be disabled.
```

Comment:
This is somewhat wrong too, its not the type, but the property name(s).
From collections, a property gets its configuration from first one that has its name in `properties`.

So you could do `colors.properties = []`.

Another thought about this. Maybe we should reverse the order of processing, so the last added items would be considered first. (I'm not sure if this applies to all javascript engines though)

Reply:
> From collections, a property gets its configuration from first one that has its name in `properties`.

Ok! I can add this note as well.

> Another thought about this. Maybe we should reverse the order of processing, so the last added items would be considered first. (I'm not sure if this applies to all javascript engines though)

I think it make sense